### ISSUE TITLE
Error when deleting stale repo

### ISSUE THREAD

**Issue by KonradStanski** [sourcebot-dev/sourcebot#130]
```
[dev:backend] 2024-12-14T08:37:56.604Z info: [main] Discovered 13 unique repositories from config.
[dev:backend] 2024-12-14T08:37:57.470Z info: [main] Synced configuration file ../../config.json in 3.022s
[dev:backend] 2024-12-14T08:37:57.470Z info: [main] Deleting stale repository review.opendev.org/cfn/computing-offload:
[dev:backend] 2024-12-14T08:37:57.470Z info: [main]     Deleting git directory /Users/konrad/git/sourcebot/.sourcebot/repos/review.opendev.org/cfn/computing-offload.git...
[dev:backend] Shutting down...
[dev:backend] node:internal/process/promises:394
[dev:backend]     triggerUncaughtException(err, true /* fromPromise */);
[dev:backend]     ^
[dev:backend]
[dev:backend] [Error: ENOENT: no such file or directory, lstat '/Users/konrad/git/sourcebot/.sourcebot/repos/review.opendev.org/cfn/computing-offload.git'] {
[dev:backend]   errno: -2,
[dev:backend]   code: 'ENOENT',
[dev:backend]   syscall: 'lstat',
[dev:backend]   path: '/Users/konrad/git/sourcebot/.sourcebot/repos/review.opendev.org/cfn/computing-offload.git'
[dev:backend] }
[dev:backend]
[dev:backend] Node.js v22.12.0
[dev:backend] error Command failed with exit code 1.
[dev:backend] info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```
This was on fresh boot with .sourcebot from old version.

**Comment by brendan-kellam** [comment]
Looks to be the same issue fixed by #136 - will spin a new release tomorrow morning and then close this. Thx for reporting

**Comment by brendan-kellam** [comment]
Fixed in [v2.6.3](https://github.com/sourcebot-dev/sourcebot/releases/tag/v2.6.3)
